webrender_api/src/image.rs

```diff
@@ -222,44 +298,64 @@ pub trait BlobImageHandler: Send {
 
 /// A group of rasterization requests to execute synchronously on the scene builder thread.
 pub trait AsyncBlobImageRasterizer : Send {
+    /// Rasterize the requests.
     fn rasterize(&mut self, requests: &[BlobImageParams]) -> Vec<(BlobImageRequest, BlobImageResult)>;
 }
 
 
+/// FIXME: document
 #[derive(Copy, Clone, Debug)]
 pub struct BlobImageParams {
+    /// FIXME: document
     pub request: BlobImageRequest,
+    /// FIXME: document
     pub descriptor: BlobImageDescriptor,
+    /// FIXME: document
     pub dirty_rect: Option<DeviceUintRect>,
 }
 
+/// Backing store for blob image command streams.
 pub type BlobImageData = Vec<u8>;
 
+/// Result type for blob raserization.
 pub type BlobImageResult = Result<RasterizedBlobImage, BlobImageError>;
 
+/// Metadata (but not storage) for a blob image.
 #[repr(C)]
 #[derive(Copy, Clone, Debug)]
 pub struct BlobImageDescriptor {
+    /// Bounding rectangle of the image, in device pixels.
     pub size: DeviceUintSize,
+    /// FIXME - Document me.
     pub offset: DevicePoint,
```

Comment:
Huh! it's unused. We should probably remove it.

Reply:
It's used by wrench.